pkg/oonf_api/Makefile.include

```diff
@@ -1,5 +1,5 @@
 INCLUDES += -I$(PKGDIRBASE)/oonf_api/src-api
 
-ifeq ($(shell uname -s),Darwin)
-	CFLAGS += -Wno-reserved-id-macro -Wno-keyword-macro
+ifeq ($(TOOLCHAIN), llvm)
+    CFLAGS += -Wno-reserved-id-macro -Wno-keyword-macro -Wno-parentheses-equality
```

Comment:
What is the error you get if you don't have `-Wno-parentheses-equality` and `-Wno-reserved-id-macro`?
I only need `-Wno-keyword-macro` to build this with Clang (`clang version 4.0.0 (tags/RELEASE_400/final)`)